docs/get_started/software_support.md

```diff
@@ -1,28 +1,24 @@
-# Program Requirements
```

Comment:
Hi @philip-menzies , we would have to revert the changes here because this is a document from legal.

Reply:
No problem. Leave the language that legal provided!